src/Components/JSRootGraph.js

```diff
@@ -9,6 +9,8 @@ function createTGraphFromDataSeries(dataSeries) {
     const tgraph = JSROOT.createTGraph(dataSeries.y.length, dataSeries.x, dataSeries.y)
     tgraph.fLineColor = dataSeries.index + 1
     tgraph.fLineWidth = 2
+    tgraph.InvertBit(JSROOT.BIT(18))
```

Comment:
Add a comment with a link to JSROOT or ROOT documentation explaining what is bit 18.
Otherwise this code seems like pure magic